source/Files.cpp

```diff
@@ -59,6 +59,20 @@ namespace {
 				c = '/';
 	}
 #endif
+
+	// Open the given folder in a separate window.
+	void OpenFolder(const string &path)
+	{
+#if SDL_VERSION_ATLEAST(2, 0, 14)
+		SDL_OpenURL(("file://" + path).c_str());
+#elif defined(__linux__)
+		// Workaround for older Linux distributions that don't ship
+		// with SDL 2.0.14 yet.
+		static_cast<void>(!system(("xdg-open file://" + path).c_str()));
```

Comment:
Rather than ignore the return value, log the error, if there was one.

If there was one, it could be useful feedback to the user (and thus us by proxy) to disable the button, too.

Reply:
Good idea! But disabling probably not because it's possible for the call to succeed at a later time.

Reply:
I suggested disabling because users would ask questions like, "why does clicking this button just make it disable itself?" which would allow us to ask for user-submitted logs, rather than guess at reasons why it might fail :)
(99.9% of windows players do not launch their game from a terminal and thus have no idea when an error gets logged.)